### ISSUE TITLE
Add animated counter for total number of tales

### ISSUE THREAD

**Issue by rrcampbell-exe** [rrcampbell-exe/yarnsworths-tales#18]
## Description
Currently, the total number of tales Yarnsworth knows "pops" onto the page. Instead, it would be nice if the number counted upward to the final number in order to draw attention to it on page load.

Furthermore, a future issue might have the text of this number start smaller and grow to the `font-size` of `p` as the count increases in number, as this will make the appearance of this number less jarring on page load.

## Possible Solution
Consider revisiting the following solution on Stack Overflow: https://stackoverflow.com/questions/2540277/jquery-counter-to-count-up-to-a-target-number

**Comment by rrcampbell-exe** [comment]
This feature will likely not make it to V2 of the app once #19 is complete. It has been tagged as a `wontfix` issue for the time being.
